classes/01class/exercises/c01-aws02/README.md

```diff
@@ -1,9 +1,15 @@
-## S3(C01-E04)
+## S3 (C01-aws02)
 
-- Create a S3 bucket and upload a file to it
-- Access the file on the S3 bucket from  any the EC2 instance created in the previous exercise or from a new instance
+- Use the CLI to create a S3 bucket and upload a file to it.
+- Access the file on the S3 bucket from any the EC2 instance created in the previous exercise.
 - Submit a PR to the DevOpsAcademy repo under the `/classes/01class/exercises/c01-e04/<github-username>` folder of this class with a brief description of what you did to achieve this objective. The file containing the solution you used must be named c01-e04.txt
 
+## Submit a PR with the folowing file:
+- README.md based on the [ANSWER.md](ANSWER.md) file with the commands requested. 
+
+## References
+- [S3 Bucket Creation](https://docs.aws.amazon.com/cli/latest/reference/s3/mb.html)
+
 
 ### Tip:
 You can access the file on the S3 bucket by creating an IAM role with S3 policy and attaching it to your EC2 instance. One of the best practices when creating IAM policies is to grant least privilege to the users. The objective is to initially give the users only the access/permission they need to complete the intended task and grant additional permissions as necessary later. This is more secure than starting with permissions that are too lenient and then trying to tighten them later. 
```

Comment:
Maybe it is better to tell them which managed policy to use as the IAM class is just later in the course

Reply:
We can do it, but I reckon this part is added by one of the students in the past.
I'll make this adjustment anyway to make things clearer..

Reply:
done